build.gradle.kts

```
@@ -121,12 +128,25 @@ subprojects {
         targetCompatibility = JavaVersion.VERSION_1_8
     }
 
+    val jvmVersion = "1.8"
+
+    val compileKotlin: KotlinCompile by tasks
+    compileKotlin.kotlinOptions {
+        jvmTarget = jvmVersion
+    }
+
+    val compileTestKotlin: KotlinCompile by tasks
+    compileTestKotlin.kotlinOptions {
+        jvmTarget = jvmVersion
+    }
```

These 2 declarations look identical to me. How does it determine that one is for tests?

I'd better have it like this:
```suggestion
   tasks.withType<KotlinCompile>.configureEach {
      kotlinOptions {
          jvmTarget = JavaVersion.VERSION_1_8.toString()
      }
   }
```

---

This is the way IDEA adds Kotlin to projects in an automatic way. Delegates, if memory serve, accept the descriptor of the declaration, which includes the name. But your suggestion is more compact, indeed.